### ISSUE TITLE
The password reset not working 

### ISSUE THREAD

**Issue by albioninnovate** [hillwithsmallfields/makers#17]

**Comment by albioninnovate** [comment]
the login page (https://makers.makespace.org/users/login/) needs a link to the password reset page.  If the user has problems loging in there is no direct way to get to the reset page.

**Comment by hillwithsmallfields** [comment]
Good point, will do.  Those pages aren't in the tree that I wrote, but I
think that particular one is one that I've overridden, so I should be able
to change it easily enough.
